Docker/build-images.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Semi-nit: probably prefer "docker" to "Docker" for this directory name.  Everything else underneath the root "adelphi" directory is lower-case and I would think we'd want to preserve that here.

Reply:
Done